Assets/Scripts/Menu/MenuManager.cs

```diff
@@ -9,9 +9,6 @@
 namespace Menu {
 	public class MenuManager : MonoBehaviour
 	{
-
-		[Header("SOUND FX IN MAIN MENU")]
-		[SerializeField] AudioClip soundFx;
 		[Header("SETTINGS"), Space(3)]
 		[SerializeField] GameObject _screenMainMenu;
```

Comment:
Try to add always the access modifier. private in this case.

Serialized variable should start without _